### ISSUE TITLE
Plugin Initialization Failed (loadFromRemoteSources switch?)

### ISSUE THREAD

**Issue by SoulForgedXLVI** [RainbowMage/OverlayPlugin#59]
I recently got ACT and wanted RainbowMage's overlay plugin. I followed the regular steps of downloading, extracting, and moving the file into the ACT folder, as well as the directions on <https://github.com/RainbowMage/OverlayPlugin/blob/master/README-en.md>. I also unblocked all the .dll files and made ACT an exception on Windows Firewall. However- this message came up, and I have no idea what to do about it, as I've little knowledge about it, and understand little of what Microsoft has on guides about it, such as on <https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/loadfromremotesources-element>. It would be great to get some help- screenshot down below.
![plugininitializationfailed](https://user-images.githubusercontent.com/30130943/28142949-29d7f514-6731-11e7-9dad-842c2890f73b.PNG)

**Comment by SoulForgedXLVI** [comment]
Hmm, and I've just realized- when I executed the build.bat file it rapidly inputted several things and then closed, and I was unable to read any of it. However, I recorded this and slowed down the frames, then saw that at the bottom, in red, it said 4 Errors. Any idea as to what that might be?

**Comment by SoulForgedXLVI** [comment]
![image](https://user-images.githubusercontent.com/30130943/28143253-9c030c68-6732-11e7-83d5-8114ca248056.png)

**Comment by SoulForgedXLVI** [comment]
![image](https://user-images.githubusercontent.com/30130943/28143290-ca156bc8-6732-11e7-9aab-e0dce2af6381.png)

**Comment by hibiyasleep** [comment]
1. right-click on DLL or it's parent directory, open Properties, click `Unlock`. by default files from internet are 'blocked'.
2. run your `cmd` in administrator, as destination folder seems permitted for admin only.

**Comment by keanecat** [comment]
Unlock the zip file before you unzip it, that will make your life easier

**Comment by Montrazul** [comment]
Same problem here and the provided solutions from @hibiyasleep  and @keanecat  are not working.

**Comment by Cope93** [comment]
did u guys try to remove the "read-only" checkbox on the overlayplugin folder? it brings me 1 step further but now it crashes when it says initialized

**Comment by rayd99** [comment]
Same problem. Folder is unblocked, not read only, not in a protected location. Get the same window as above. Downloaded OverlayPlugin 0.3.3.8x64-full, running in ACT 3.3.1.255 on Windows 10. Any help would be appreciated.

**Comment by soncharmander** [comment]
I'm having the exact same issue
I have ACT installed in my F:\Program Files (x86)\Advanced Combat Tracker
I have OverlayPlugin.dll unblocked and sitting right next to my FF plugin dll
There is no "Read only" option checked

I am getting the sandbox error still.

**Comment by soncharmander** [comment]
what am I supposed to run in cmd as admin?

**Comment by soncharmander** [comment]
UPDATE:
I read through your EN readme, and I found out that my OverlayPlugin.dll was not read only, but the parent folder was
My mod is working cleanly now.

**Comment by ctelroy** [comment]
@hibiyasleep 
Right-click on **"Zip"** file, open Properties, click "Unlock" works for me.
